src/app/text-compare/text-compare.component.css

```diff
@@ -0,0 +1,26 @@
+#editor1,
+#editor2 {
+    height: 500px;
+}
+
+.editor-container {
+    border: 1px solid #ddd;
+}
+
+.input-title {
+    text-align: center;
+    color: #585858;
+    font-weight: bold;
+    text-transform: uppercase;
+    font-size: 12px;
+    border: 1px solid #ccc;
+    background-color: #ecebeb;
+}
+
+.input-title label {
+    margin-top: 6px;
+}
+
+.diff-editor {
+    height: 40vh !important;
```

Comment:
Is it smart to use `vh` here and not a more precise measure or `%`?

Reply:
Yeah the idea is for the diff editor to take 40% of the height available in the visible window for the user